structured_external_storage_all/models/ir_attachment.py

```diff
@@ -2,47 +2,37 @@
 # © 2018 Sunflower IT (http://sunflowerweb.nl)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 import logging
-import functools
 import datetime
 import dateutil.relativedelta as relativedelta
 from odoo import api, models, tools
 from odoo.tools.safe_eval import safe_eval
 from werkzeug import urls
-
+from jinja2.sandbox import SandboxedEnvironment
 _logger = logging.getLogger(__name__)
-try:
-    from jinja2.sandbox import SandboxedEnvironment
-    mako_template_env = SandboxedEnvironment(
-        block_start_string="<%",
-        block_end_string="%>",
-        variable_start_string="${",
-        variable_end_string="}",
-        comment_start_string="<%doc>",
-        comment_end_string="</%doc>",
-        line_statement_prefix="%",
-        line_comment_prefix="##",
-        trim_blocks=True,               # do not output newline after blocks
-        autoescape=True,                # XML/HTML automatic escaping
-    )
-    mako_template_env.globals.update({
-        'str': str,
-        'quote': urls.url_quote,
-        'urlencode': urls.url_encode,
-        'datetime': datetime,
-        'len': len,
-        'abs': abs,
-        'min': min,
-        'max': max,
-        'sum': sum,
-        'filter': filter,
-        'reduce': functools.reduce,
-        'map': map,
-        'round': round,
-
-        'relativedelta': lambda *a, **kw: relativedelta.relativedelta(*a, **kw)
-    })
-except ImportError:
-    _logger.warning("jinja2 not available, templating features will not work!")
+jinja2_template_env = SandboxedEnvironment(
+    block_start_string="<%",
+    block_end_string="%>",
+    variable_start_string="${",
+    variable_end_string="}",
+    comment_start_string="<%doc>",
+    comment_end_string="</%doc>",
+    line_statement_prefix="%",
+    line_comment_prefix="##",
+    trim_blocks=True,               # do not output newline after blocks
+    autoescape=True,                # XML/HTML automatic escaping
```

Comment:
Remind that these ones could all go if we switch to Jinja2 syntax instead of Mako.